projects/pastanaga-angular/src/lib/controls/textfield/select/select.component.ts

```diff
@@ -203,6 +203,9 @@ export class SelectComponent extends PaFormControlDirective implements OnChanges
     selectOption(option: OptionModel | OptionComponent) {
         if (!option.disabled && this.isActive && option.value !== this.control.value) {
             this.control.patchValue(option.value);
+            if (!!this.selectInput) {
+                this.selectInput.nativeElement.click();
+            }
```

Comment:
I don't think that the right way of solving this bug as clicking on the input might have some side effect if there is a listener on click event set on the select.